src/geometries.jl

```diff
@@ -130,56 +148,47 @@ See https://en.wikipedia.org/wiki/Facet_(geometry)
 function facets end
 
 """
-    center(polytope)
+    p1 == p2
 
-Return the center of the `polytope`.
+Tells whether or not polytopes `p1` and `p2` are equal.
 """
-center(p::Polytope) = Point(sum(coordinates.(p.vertices)) / length(p.vertices))
+==(p1::Polytope, p2::Polytope) = p1.vertices == p2.vertices
```

Comment:
I am still unsure about testing for equality only on vertices, and using `foo.vertices` to do that. It won't work for some polytopes such as `PolySurface`.
We can always add an equality method for `PolySurface`s though. But a `PolySurface` instance can represent exactly the same thing as a `Triangle` with the right parameters, making identical polytopes. But I guess it's not the place to discuss that.

Reply:
Yes, I agree we can brainstorm the equality of polytopes in a separate issue. I think it will be natural to discuss it after we polish some other pressing details first like the ordering issue, and the representation of polytopes in general.